hivemind/proto/crypto.proto

```diff
@@ -0,0 +1,24 @@
+//Originally taken from: https://github.com/mhchia/py-libp2p-daemon-bindings
+//Licence: MIT
+//Author: Kevin Mai-Husan Chia
```

Comment:
Can you please clarify that this is taken from [py-libp2p](https://github.com/libp2p/py-libp2p/blob/master/libp2p/crypto/pb/crypto.proto) (not the p2p bindings themselves)?

This is not only to acknowledge the authors, but to make it clear where the code comes from and why it's written this way.